publish.js

```diff
@@ -0,0 +1,33 @@
+const fs = require('fs');
```

Comment:
misschien wat uitleg verschaffen rond de raison d'être van dit bestand in een comment? anders kan een overijverig iemand het wegsmijten